CMakeLists.txt

```diff
@@ -171,7 +171,7 @@ therock_add_feature(PRIM
 therock_add_feature(BLAS
   GROUP MATH_LIBS
   DESCRIPTION "Enables blas libraries (hipblaslt, rocblas, hipblas)"
-  REQUIRES COMPILER HIP_RUNTIME
+  REQUIRES COMPILER HIP_RUNTIME HOST_BLAS PROFILER_SDK
```

Comment:
`HOST_BLAS` or rather `therock-host-blas` is an optional dep only required when building tests. Should be add here nevertheless?

Reply:
Let me add a note to that effect for the future. I added this because I was doing a subset build and it was broken otherwise.